src/resolver.rs

```diff
@@ -179,7 +175,9 @@ impl DependencyGraph {
                 })?;
 
             let package = registry
-                .download(dependency.with_version(&local_locked.version))
+                // fetch the version using manifest directly. This works now
+                // because buffrs only supports pinned versions
```

Comment:
I mean technically this is okay, since the entire version resolution is subject to change soon

Reply:
Nonetheless its a bit dangerous 😁

Reply:
Do you want me to add a TODO here to remember this whenever dynamic versioning is implemented?